WeakAurasOptions/TriggerOptions.lua

```diff
@@ -253,10 +253,34 @@ function OptionsPrivate.ClearTriggerExpandState()
   maxTriggerNumForExpand = 0
 end
 
+function OptionsPrivate.GetTriggerTitle(data, triggernum, prefix)
```

Comment:
prefix appears to be always set to true.

Reply:
Indeed, i wasn't sure i would have always use a prefix, at first, argument removed.